concrete/src/Page/Collection/Collection.php

```diff
@@ -378,15 +357,15 @@ public function loadVersionObject($cvID = 'ACTIVE')
      */
     public function getVersionToModify()
     {
-        $u = new User();
+        if ($this->isMasterCollection()) {
+            return $this;
+        }
         $vObj = $this->getVersionObject();
-        if ($this->isMasterCollection() || ($vObj->isNew())) {
+        if ($vObj->isNew()) {
```

Comment:
We could have a `null` version object for instance when calling `Collection::getByID(1, false)`.
Sure, we could always throw an exception when trying to access a `null` version (so that it can be catched by PHP 5 - in PHP 7+ we can intercept `Error` exceptions)